### DISCUSSION TITLE
TSML-UI: Change Text "Location Temporarily Closed"

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#781] [Q&A] [answered]
What is the status of this request? [See #40
](https://github.com/code4recovery/spec/discussions/40)

Would it be possible to change Location Temporarily Closed to “Not Meeting at Physical Location” or “Meeting Online Only” or something like that? This has nothing to do with suspended meetings. It only affects meetings that are marked as "tc" that are meeting online.

**Comment by joshreisner** [comment, marked as answer]
we're waiting to get alignment with GSO to make a global change. i'll make sure it's on the agenda to discuss at our next meeting.

i think you should be able to make this change locally with 

```
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'TC' => 'Not Meeting at Physical Location',
    ));
}
```

**Reply by joshreisner** [reply]
based on GSO's response of "not at this time" i'm going to mark this issue as "answered"
